applications/static/main.tf

```diff
@@ -66,12 +124,3 @@ resource "aws_s3_bucket_policy" "bucket_policy" {
   bucket = aws_s3_bucket.bucket.id
   policy = data.template_file.public_bucket_policy.rendered
 }
-
-output "domain" {
-  value = var.domain
-}
-
-output "backend" {
-  value = "${aws_s3_bucket.bucket.id}.s3-website-${aws_s3_bucket.bucket.region}.amazonaws.com"
-}
-
```

Comment:
Since we consume the S3 bucket "backend" inside the module, it no longer makes sense to output these.